text/chapter2.md

```diff
@@ -52,12 +52,8 @@ Let's work through this next exercise together in test-driven-development style.
 
 We'll start by enabling the tests for this exercise. Move the start of the block-comment down a few lines as shown below. Block comments start with `{-` and end with `-}`:
 ```hs
-    suite "diagonal" do
-      test "3 4 5" do
-        Assert.equal 5.0 (diagonal 3.0 4.0)
-      test "5 12 13" do
-        Assert.equal 13.0 (diagonal 5.0 12.0)
-{-  Move this block comment starting point to enable more tests
+{{#include ../exercises/chapter2/test/Main.purs:diagonalTests}}
+    {-  Move this block comment starting point to enable more tests
```

Comment:
Is this preferred, or would it be better to leave this with no anchor at all since we have to "modify" it by adding this line at the end?

Reply:
I think your version looks good. I believe this combo technique was used in some other places too in the book.